docs/communication/index.md

```diff
@@ -27,4 +27,4 @@ For more information:
 - [Fearless feedback](fearless-feedback.md)
 - [Shoutouts](fearless-feedback.md#shoutouts)
 - [Scenarios](scenarios.md)
-- Writing Style Guide
+- [Writing Style Guide](docs/../../processes/documentation/writing-style-guide.md)
```

Comment:
**suggestion:** Could we use a relative link?

```suggestion
- [Writing Style Guide](../processes/documentation/writing-style-guide.md)
```

Reply:
Can do!